_includes/ecosystem_form.html

```diff
@@ -0,0 +1,71 @@
+<div class="col-md-12">
+  <form class="ecosystem-form" action="https://twitter.us14.list-manage.com/subscribe/post" method="POST">
```

Comment:
I updated this form on Mailchimp so that everything is required. you might need to repaste it into here